network/network.go

```diff
@@ -53,6 +53,24 @@ const (
 	CannotConnect
 )
 
+// Routability indicates how reachable a node is.
+type Routability int
```

Comment:
worth making this a `byte`?

Reply:
@willscott What's the advantage of making this a byte ? Saving space ?

Reply:
> @willscott What's the advantage of making this a byte ? Saving space ?

Right. Could potentially allow for better packing

Reply:
I'd support it.

Reply:
No real reason not to.

Reply:
Hm, well, except for consistency with `Connectedness` and most other enums.